opal/mca/btl/ofi/btl_ofi_providers.h

```diff
@@ -0,0 +1,42 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
+/*
+ * Copyright (c) 2018      Intel, Inc, All rights reserved
+ * Copyright (c) 2018      Los Alamos National Security, LLC. All rights
+ *                         reserved.
+ *
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "btl_ofi.h"
+
+#define MCA_BTL_OFI_REQUIRED_CAPS       (FI_RMA | FI_ATOMIC)
+#define MCA_BTL_OFI_REQUESTED_MR_MODE   (FI_MR_UNSPEC)
+
+
+/* validate information returned from fi_getinfo().
+ * return OPAL_ERROR if we dont have what we need. */
+static int validate_info(struct fi_info *info)
+{
+    int mr_mode;
+
+    /* we need exactly all the required bits */
+    if ((info->caps & MCA_BTL_OFI_REQUIRED_CAPS) != MCA_BTL_OFI_REQUIRED_CAPS)
+        return OPAL_ERROR;
```

Comment:
Nit pick on the style: don't forget to `{}` *all* blocks, even 1-line blocks.  Sorry -- that's OMPI style.  😉

Reply:
I tried to catch all of it. Seems like I missed this one. I got confused midway between libfabric and ompi standard.